src/mcode-pilot/services/outcomes/CLQOutcomesService.js

```diff
@@ -183,9 +184,11 @@ export default class CLQOutcomesService extends IOutcomesService {
     response into a format that is used by the Compass UI */
     processSimilarPatientOutcomes(similarPatientProps) {
         let filter = {};
-        filter.demographics = this.buildDemographicsFilter(similarPatientProps);
-        filter.diagnosis = this.buildDiagnosisFilter(similarPatientProps);
-        filter.tumorMarkers = this.buildTumorMakersFilter(similarPatientProps);
+        const fOptions = new FilterOptions(similarPatientProps);
```

Comment:
I think it would be better to have the FilterOptions passed into the method instead of creating the class within the method.